src/module/DiffusionSDE.cpp

```diff
@@ -96,33 +111,47 @@ void DiffusionSDE::process(Candidate *candidate) const {
 	  	propTime *= 0.5;
 		counter += 1;
 
-    //Check for better break condition
-    // Check zero magnetic field
+    // Check for better break condition
 	} while (r > 1 && fabs(propTime) >= minStep/c_light); 
+	
 	//std::cout << "r = " << r << "\n";
 	//std::cout << "propTime = " << propTime*c_light/kpc << "\n";
 	//std::cout << "counter = " << counter << "\n";
 	
```

Comment:
Commented print lines should be removed (if we don't do it now, it will stay here forever). If there is a need for it use KiSS macros.